### ISSUE TITLE
KeyError: 'camera_type'

### ISSUE THREAD

**Issue by charlesmillet** [Stefal/mapillary_download#8]
Downloading stops on some sequences, on a specific photo it seems. I tried to restart the download and sometime it download some additional photos on the first attempt but then the download stops, maybe not always on the same photo cause the download doesn't seem to follow an order.
The message is the following:
```
Traceback (most recent call last):
  File "mapillary_download.py", line 162, in <module>
KeyError: 'camera_type'
[21287] Failed to execute script 'mapillary_download' due to unhandled exception!

```
It appends on several sequences like this one: a1eg384w8wcw1ybyw3shju

**Comment by Stefal** [comment]
The `a1eg384w8wcw1ybyw3shju` sequence include some images without `camera_type` value.

**Comment by Stefal** [comment]
New release available (v1.5)
